jenkins/jobs/cd/build_tiflash/master.groovy

```diff
@@ -21,7 +21,7 @@ pipelineJob('build-tiflash-master') {
         }
     }
     parameters {
-        stringParam('GitHash', '7bd02a86e08f5077c69299ac048a700fcb68507f', 'the git tag or commit')
+        stringParam('GitHash', '', 'the git tag or commit')
         stringParam('Version', 'v7.3.0', 'important, the Version for cli --Version and profile choosing, eg. v6.5.0')
```

Comment:
Suggested `Version` default value is kept as an empty string.

Reply:
fixed